pipeline/mutate/mutator_hydrator.go

```diff
@@ -114,16 +114,16 @@ func (a *MutatorHydrator) GetID() string {
 	return "hydrator"
 }
 
-func (a *MutatorHydrator) cacheKey(config *MutatorHydratorConfig, session string) string {
-	return fmt.Sprintf("%s|%x", config.Api.URL, md5.Sum([]byte(session)))
+func (a *MutatorHydrator) cacheKey(config *MutatorHydratorConfig, key string) string {
+	return fmt.Sprintf("%s|%x", config.Api.URL, md5.Sum([]byte(key)))
```

Comment:
The key is taken as a literal and not being executed. So `{{ print .Subject }}` would just be interpreted as a literal string, not the actual subject ID :)

If you want to execute it, you need to render the template and give it the data.

Reply:
Done.